spec/openapi.yaml

```diff
@@ -26,6 +26,19 @@ info:
     The MessageMedia APIs provide powerful business messaging capabilities across sending, receiving, and processing SMS, MMS, and rich messaging. 
 
     All requests to the MessageMedia REST API must be authenticated, this can either be done using Basic Authentication or by signing with a HMAC signature.
+    
+    *Note: MMS and TTS are not supported for the EU base url: https://eu.app.api.sinch.com*
+
+
+    # Base URI
+      
+    The API uses the following base URI
+    
+    For EU instance please use eu.app.api.sinch.com
+    
+    For APAC instance, use api.messagemedia.com
+    
+    *Note: MessageMedia strongly recommends the use of the HTTPS protocol.*
```

Comment:
Do we need to mention this? Maybe just update the URL above to include the https?